### ISSUE TITLE
Issue with searching for movies.

### ISSUE THREAD

**Issue by AdeolaFaraday** [Go-phie/gophie-web#111]
Movies are not fetched when searched, takes to long or probably won't fetched at all.

**Comment by deven96** [comment]
> Movies are not fetched when searched, takes to long or probably won't fetched at all.

Yeah we are aware of the issue and are working on a fix. This will be closed when it works
